daml-lf/transaction/src/main/protobuf/com/daml/lf/transaction.proto

```diff
@@ -126,4 +126,17 @@ message NodeRollback { // *since version 14*
     repeated string children = 1; // node ids
 }
 
+message FatContractInstance {
+  bytes contract_id = 1;
+  com.daml.lf.value.Identifier template_id = 2;
+  bytes arg = 3;
```

Comment:
As I understand this is a field representing contract_arguments? I would call it that then
Also, do we need the bytes at this level? We already have 2 levels of proto encoding around it: 1) ledger-api 2) versioned blob.

Reply:
> As I understand this is a field representing contract_arguments? I would call it that then